src/dmd/expression.d

```diff
@@ -4769,8 +4769,6 @@ extern (C++) final class DotVarExp : UnaExp
     override Modifiable checkModifiable(Scope* sc, int flag)
     {
         //printf("DotVarExp::checkModifiable %s %s\n", toChars(), type.toChars());
```

Comment:
Remove the printf statement?

Reply:
No, it was already there and is sometimes useful while debugging.